jobs/warcorrespondent.md

```diff
@@ -0,0 +1,25 @@
+## Abstract
+
+War Correspondent is a journalist role, meant to be a sort've independent/marine aligned CL. It's meant to provide more roleplay opportunities and add more flavour. It provides more opportunities because it's contextually different from CL who is usually maligned by Marine players, with this role it'll be completely different as WC starts with 0 W-Y affiliation and probably won't get such derision, along with just being a contextually different role with different objectives than the CL. 
+
+## Goals
+
+1. Create a role that's interesting and provides new oppirtunities along with spicing up the game, the WC is contextually different from any other role in the game, providing leeway for players to get creative with the way they play their character in this role.
+2. Create a roleplay-centric role that's differentiated from the CL enough to be its own thing
+
+## Non-goals
+
+1. Making another CL, the CL has a specific role as the W-Y guy. This role should not just be just that, it's contextually different and provides players a choice with who they align with.
+2. Shifting the balance of the game, WC has no impact on the general balance of the game, and shouldn't. It gets the same gear as a PFC would.
+3. Giving new opportunities for grief
+
+## Content
+
+Roleplay role similar to CL but gets their own vendor and armor for deployment, similar to PFC in that way but is contextually different and not provided enough to be a complete match for PFC, leading to more roleplay and less PFC'ing. TLDR it's designed to be like an actual War Correspondent, not focused on shooting things but rather doing Journalism, still being provided with tools to defend themselves
```

Comment:
need an empty line after this otherwise it messes up the formatting. do the same for all the other categories

Reply:
ideally this should also be more detailed, explain what sort of equipment they can get and what sort of gear they're given. Remove the TLDR and integrate it into your explanation